pkgs/by-name/so/sobjectizer/package.nix

```diff
@@ -0,0 +1,59 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  callPackage,
+  buildExamples ? false,
+  buildTests ? false,
+  buildAll ? false,
+}:
```

Comment:
```diff
+ withStatic ? stdenv.hostPlatform.isStatic,
+ withShared ? !stdenv.hostPlatform.isStatic
```

Reply:
> ```diff
> + withStatic ? stdenv.hostPlatform.isStatic,
> + withShared ? !withStatic
> ```

Static library(and shared too) is required for building examples, tests, and for build-all option.
From what I gather, `stdenv.hostPlatform.isStatic` is false in common case. So to be able to build tests & co we will need set `withStatic` manually.

Can we mark withStatic(withShared too) as true by default or there is another approach to resolve this?

Thanks.

Reply:
Hm, I see. What an interesting test setup 😄 you could set `doCheck = withStatic && withShared`.
This would play together nicely with `(lib.cmakeBool "BUILD_TESTS" finalAttrs.doCheck)`